packages/terra-clinical-data-grid/tests/wdio/data-grid-spec.js

```diff
@@ -0,0 +1,4 @@
+
+
```

Comment:
whoops. Coming?

Reply:
Yep, not trying to pull a fast one on you. The only thing I dislike more than writing tests is updating tests once they're already written :)

I'll write these when we have all the final input from Neil on themeable properties and everything.